Code/Source/URDF/Constants.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <string>
+
+namespace ROS2
+{
+    ////////////////////////////////////////////////////////////////////////////////////////////////////
+    //! The struct contains all constant parameters used in FBX files.
+    //! TODO: Could be moved to configuration file
+    namespace Fbx
+    {
+        struct Constants
+        {
+            struct FbxHeader
+            {
+                inline static const int headerVersion = 1003;
```

Comment:
```inline``` is not necessary, and you can use constexpr instead for all the constants
```suggestion
                static constexpr int headerVersion = 1003;
```

Reply:
IMHO not a big difference but yes, probably `constexpr` a bit better expresses the intention of compile-time constant usage.